reportTemplates.json

```diff
@@ -10678,7 +10678,7 @@
 					"inputs": [
 						{
 							"name": "title",
-							"format": "COLLABORATIVE EXPLORATION INITIATIVE [title] - [permit] FINAL REPORT",
+							"format": "COLLABORATIVE EXPLORATION INITIATIVE [title] - [permit]",
 							"viewLabel": "Title of the Report",
 							"label": "ROUND # - CEI# PROJECT NAME - CEI ACTIVITY",
```

Comment:
Stumbled on a fairly trivial inconsistency here... The label and the placeholder are inconsistent (lines 10683-10684)
Do we want a - or a : in the title?